projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts

```diff
@@ -207,9 +207,11 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
       case FilterOperator.Equals:
         return rows.filter(eachRow => ((eachRow[actualKeyNameInRows] as Entity).name as string) === tableFilter.value);
       case FilterOperator.Like:
-        const regexExp = new RegExp(tableFilter.value as string);
+        const regexExp = new RegExp((tableFilter.value as string).toLowerCase());
 
-        return rows.filter(eachRow => regexExp.test((eachRow[actualKeyNameInRows] as Entity).name as string));
+        return rows.filter(eachRow =>
+          regexExp.test(((eachRow[actualKeyNameInRows] as Entity).name as string).toLowerCase())
```

Comment:
The lowerCase transform wouldn't be needed if we configure the regex to ignore case.